modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -121,9 +119,7 @@ const DocumentLibrary = ({
 							disabled={readOnly}
 							id={`${name}inputFile`}
 							lang={editingLanguageId}
-							onBlur={onBlur}
 							onClick={onSelectButtonClicked}
-							onFocus={onFocus}
```

Comment:
Hey @natocesarrego,
since `onSelectButtonClicked` has the logic of that and we pass it through both of select button and input, we don't need to pass onFocus and onBlur through input :smile